content/manuals/engine/install/debian.md

```diff
@@ -110,8 +110,8 @@ Docker from the repository.
    sudo apt-get update
    sudo apt-get install ca-certificates curl
    sudo install -m 0755 -d /etc/apt/keyrings
-   sudo curl -fsSL {{% param "download-url-base" %}}/gpg -o /etc/apt/keyrings/docker.asc
-   sudo chmod a+r /etc/apt/keyrings/docker.asc
+   sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor --yes -o /etc/apt/keyrings/docker.gpg
+   sudo chmod a+r /etc/apt/keyrings/docker.gpg
```

Comment:
1. This adds the Ubuntu key, but this is the instruction for Debian. We should use the Debian key here.
2. APT supports `asc`, see #19138

Reply:
With the `.asc` file I can not use the docker repo:
```
root@ligneris-cloud:~# ls -la /etc/apt/keyrings/
total 12
drwxr-xr-x 2 root root 4096 Sep 27 07:54 .
drwxr-xr-x 9 root root 4096 Sep 27 07:54 ..
-rw-r--r-- 1 root root 3817 Sep 26 21:57 docker.asc
root@ligneris-cloud:~# apt update
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 https://download.docker.com/linux/debian bookworm InRelease                  
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease                      
Hit:4 http://security.debian.org/debian-security bookworm-security InRelease
Err:2 https://download.docker.com/linux/debian bookworm InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7EA0A9C3F273FCD8
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: https://download.docker.com/linux/debian bookworm InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7EA0A9C3F273FCD8
W: Failed to fetch https://download.docker.com/linux/debian/dists/bookworm/InRelease  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7EA0A9C3F273FCD8
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

Reply:
With the `gpg` file (from Ubuntu) it works:

```root@ligneris-cloud:/etc/apt/keyrings# ls -la /etc/apt/keyrings/docker.gpg 
-rw-r--r-- 1 root root 2760 Sep 26 21:59 /etc/apt/keyrings/docker.gpg
root@ligneris-cloud:/etc/apt/keyrings# apt update
Hit:1 http://security.debian.org/debian-security bookworm-security InRelease
Hit:2 http://deb.debian.org/debian bookworm InRelease                          
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease                  
Hit:4 https://download.docker.com/linux/debian bookworm InRelease
Reading package lists... Done                       
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
```

Reply:
Good catch regarding `debian` vs `ubuntu`.

I believe we should change the download to:
`sudo curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor --yes -o /etc/apt/keyrings/docker.gpg`

It happens to be the exact same signing key but it could change in the future

I tested it locally and they are the same keys...

```
ls -la
total 16
drwxr-xr-x 2 root root 4096 Sep 27 08:00 .
drwxr-xr-x 9 root root 4096 Sep 27 07:58 ..
-rw-r--r-- 1 root root 2760 Sep 27 08:00 debian.gpg
-rw-r--r-- 1 root root 2760 Sep 26 21:59 docker.gpg
root@ligneris-cloud:/etc/apt/keyrings# diff docker.gpg debian.gpg 
```